src/components/SNTLmap.vue

```diff
@@ -1790,25 +1796,23 @@ export default {
 </script>
 <style lang="scss" scoped>
 .map-grid{
+  display: grid;
+  grid-template-columns: repeat(6, 1fr);
+  grid-template-areas: 
+    ". legend legend legend legend ."
+    "ak ak ak ak ak ak"
+    "us us us us us us"
+    "line line line line line line"
```

Comment:
that's awesome, I didn't know about `grid-template-areas` but it seems like the ideal solution